### PR TITLE
rating mentor

### PR DESCRIPTION
**Pull request by wowKimsohyun17**
## Related Tickets:
- [#30241](https://edu-redmine.sun-asterisk.vn/issues/30241)

## WHAT this PR does?
rating mentor

## Notes Impacted Areas:
- N/A.

## Attachments:
hehe

### CODE REVIEW

**Comment by wowKimsohyun17** [comment]
ready

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -17,6 +18,15 @@ class HomeController extends Controller
...
         $courses = Course::with('image')->latest('created_at')->take(config('title.course_number'))->get();
+        if (Auth::check()) {
+            $requestsOfUser = Advisor::whereNotNull('mentor_id')
+                                    ->where([['student_id', Auth::id()], ['status', config('status.request.pending_number')]])
+                                    ->with('mentor.image', 'lesson.course')
+                                    ->get();
+
+            return view('user.component.index', compact('courses', 'requestsOfUser'));
+
+        }
```
Comment (huyenttm-0901) [inline app/Http/Controllers/HomeController.php]: convention (luom)

File: app/Http/Controllers/User/StudentController.php [hunk 2]
```diff
@@ -197,4 +198,29 @@ public function getExercisesOfUserByLessonId($id)
...
     }
+
+    public function storeRating(StoreRatingRequest $request, User $mentor)
+    {
+        // dd($request->all());
+        // $mentor->mentorComments()->create([
+        //     'content' => $request->content,
+        //     'rate' => $request->rate,
+        //     'user_id' => Auth::id(),
+        // ]);
```
Comment (huyenttm-0901) [inline app/Http/Controllers/User/StudentController.php]: ko dùng xóa đi

**Comment by wowKimsohyun17** [comment]
ready

**Pull request merged**